telematic_system/telematic_historical_data_processing/rosbag2_processing_service/rosbag2_processing_service/rosbag_processor.py

```diff
@@ -80,7 +82,17 @@ def process_rosbag(self,rosbag2_path):
 
         # Load the rosbag from the config directory
         try:
-            for msg in read_ros2_messages(rosbag2_path):
+            fd = open(rosbag2_path, "rb")
```

Comment:
can we replace this with "with resource open" so we do not need to close it manually?